app/persistence/fabric/CRUDService.ts

```diff
@@ -540,6 +564,22 @@ export class CRUDService {
 	}
 	// Orderer BE-303
 
+	/**
+	 *
```

Comment:
Please add what this function returns here for better understanding.

Reply:
Added what the function is returning as mentioned.